scripts/deeplink-dispatcher.sh

```
@@ -0,0 +1,316 @@
+    *)
+      echo "Invalid selection. Please try again."
+      send_link "$link"
+      ;;
+  esac
+}
+
+# Function to handle sending link to iOS
+handle_ios() {
+  local link="$1"
+  local ios_sims
+  ios_sims=($(detect_ios_simulators))
```

Shellcheck

[warning] Prefer mapfile or read -a to split command output (or quote to avoid splitting).

(SC2207)